sources/lib/util/steps.ml

```diff
@@ -0,0 +1,73 @@
+open Options
+open Format
+
+type incr_kind = Matching | Omega | Fourier | Uf | Builtin | Ac | Naive of int
+
+let naive_steps = ref 0
+let steps = ref 0
+let mult_f = ref 0
+let mult_m = ref 0
+let mult_s = ref 0
+let mult_uf = ref 0
+let mult_b = ref 0
+let mult_a = ref 0
+
+let incr k =
```

Comment:
Maybe add a comment stating the the "mutlipliers" are here to homogeneize the global step counter by (strangely enough) dividing each step kind by its multiplier.